Userland/Libraries/LibSQL/Value.cpp

```diff
@@ -143,30 +155,30 @@ DeprecatedString Value::to_deprecated_string() const
         });
 }
 
-Optional<int> Value::to_int() const
+Optional<i64> Value::to_i64() const
 {
     if (is_null())
         return {};
 
     return m_value->visit(
-        [](DeprecatedString const& value) -> Optional<int> { return value.to_int(); },
-        [](int value) -> Optional<int> { return value; },
-        [](double value) -> Optional<int> {
-            if (value > static_cast<double>(NumericLimits<int>::max()))
+        [](DeprecatedString const& value) -> Optional<i64> { return value.to_int<i64>(); },
+        [](i64 value) -> Optional<i64> { return value; },
+        [](double value) -> Optional<i64> {
+            if (value > static_cast<double>(NumericLimits<i64>::max()))
                 return {};
-            if (value < static_cast<double>(NumericLimits<int>::min()))
+            if (value < static_cast<double>(NumericLimits<i64>::min()))
                 return {};
-            return static_cast<int>(round(value));
+            return static_cast<i64>(round(value));
         },
-        [](bool value) -> Optional<int> { return static_cast<int>(value); },
-        [](TupleValue const&) -> Optional<int> { return {}; });
+        [](bool value) -> Optional<i64> { return static_cast<i64>(value); },
+        [](TupleValue const&) -> Optional<i64> { return {}; });
 }
 
-Optional<u32> Value::to_u32() const
+Optional<u64> Value::to_u64() const
 {
     // FIXME: Handle negative values.
-    if (auto result = to_int(); result.has_value())
-        return static_cast<u32>(result.value());
+    if (auto result = to_i64(); result.has_value())
+        return static_cast<u64>(result.value());
```

Comment:
I know this issue is already existing but negative now produce huge `u64`s which seems like it would be less expected than just bailing at this point

But :shrug: I guess we also wipe the db if the version changes so if we treat LibSQL as on the bleeding edge then it's fine to keep this

Reply:
We now bail anytime a requested conversion isn't possible due to the current value not fitting in the requested type (e.g. converting a negative number be converted to an unsigned type).